ballerina-tests/tests/48_bidi_timestamp_service.bal

```diff
@@ -23,8 +23,8 @@ listener grpc:Listener ep48 = new (9148);
 service "BidiStreamingTimestampService" on ep48 {
 
     remote function bidiStreamingGreetServer(BidiStreamingTimestampServiceBiDiGreetingCaller caller,
-     stream<string, error?> clientStream) returns error? {
-        check clientStream.forEach(function(string value) {
+    stream<string, error?> clientStream) {
+        checkpanic clientStream.forEach(function(string value) {
```

Comment:
```suggestion
        check clientStream.forEach(function(string value) {
```

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/606/commits/c32ef7e98e2bfaa4df966a96025d16d755dfb1cd